src/subscriptions/SubscriptionManager.cpp

```diff
@@ -245,8 +243,8 @@ SubscriptionManager::subBook(
     std::shared_ptr<WsBase> session)
 {
     subscribeHelper(
-        session, book, bookSubscribers_, [this, book](session_ptr session) {
-            unsubBook(book, session);
+        session, book, bookSubscribers_, [this, book](session_ptr sesh) {
```

Comment:
wonder after u remove the lambda parameter , the function parameter session is not used. does it throw warning?

Reply:
@cindyyan317 It will throw compilation error unless I fundamentally modfiy subscribeHelper in SubscriptionManager.h, as it does not match the format.

Upon modifying subscribeHelper to get rid of this, however, introduced bug chains as this parameter was utilized elsewhere within SubscriptionManager.cpp. As a general rule, I think it is better to keep these parameters, despite being unused, so that bug chains are not unknowingly created in the future + these parameters can be utilized in the future need be 

![image](https://user-images.githubusercontent.com/47958858/210519443-57259056-18d1-41af-9faf-3c79cc7aef9a.png)
![image](https://user-images.githubusercontent.com/47958858/210519823-5d7552c0-71d5-4c96-8964-ac9ef2576c91.png)